docs/operator-guide/cd/deploy-rpm.md

```diff
@@ -0,0 +1,310 @@
+
+In KubeRocketCI, RPM support allows to collect applications, store them as Nexus artifacts, and deploy them using [Ansible](https://docs.ansible.com/ansible/latest/index.html) tool.
+RPM package support implementation is implemented in two approaches:
+
+1. **Default approach**: This approach implies configuring deployments using GitOps repository and Kubernetes secrets. It is considered as more complicated but flexible at the same time.
+2. **AWX approach**: This approach implies deploying applications using [AWX](https://ansible.readthedocs.io/projects/awx/en/latest/) tool.
+
+## Default Approach
+
+The first approach to use RPM packages is Default. This approach is considered as more complicated but efficient at the same time.
+
+### Integration Flow
+
+Default approach involves the following steps:
+
+```mermaid
+graph LR;
+    A(Onboard GitOps repository) --> B(Create GitOps repository secret) --> C(Create host secret) --> D(Create application)
+```
+
+Here is a breakdown of the scheme above:
+
+1. **Onboard GitOps repository**: GitOps repository is used to store pre-deploy information (such as dependencies) and Ansible configuration.
+2. **Create GitOps repository secret**: Repository secret is required for downloading applications to your workloads.
+3. **Create host secret**: Host secret is necessary for deploy pipelines to access your workloads.
+4. **Create application**: Once all the system resources are created, proceed with creating application in KubeRocketCI portal.
+
+:::note
+  The **pre-deploy.yml** file will take affect only if the **Values override** option is enabled when deploying application.
+:::
+
+By completing these steps, you will be able to add, store, and deploy RPM applications.
+
+To set up RPM integration, follow the steps below:
+
+1. Open the KubeRocketCI portal. Navigate to **Configuration** -> **Deployment** -> **GitOps**:
+
+  ![GitOps tab](../../assets/operator-guide/rpm/ansible-add-gitops.png "GitOps tab")
+
+2. Specify the GitOps repository in the **username/repository_name** format:
+
+  ![Add GitOps repository](../../assets/operator-guide/rpm/ansible-gitops.png "Add GitOps repository")
+
+3. Create and apply the secret for the GitOps repository by running the command below:
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cd-ansible-gitops-key
+  namespace: <krci>
+data:
+  id_rsa: $(cat /path/to/repo_id_rsa | base64 | tr -d '\n')
```

Comment:
can i use these manifestos like this? is there a similar description in other guides?

Reply:
These manifests are valid, I've seen similar description in our documentation. For example:

* [Argo CD Integration](https://docs.kuberocketci.io/docs/operator-guide/cd/argocd-integration#keycloak-integration-optional)
* [Install via AWS Marketplace](https://docs.kuberocketci.io/docs/operator-guide/aws-marketplace-install#deploy-kuberocketci-platform)